src/openforms/formio/tests/validation/test_addressnl.py

```diff
@@ -0,0 +1,97 @@
+from django.test import SimpleTestCase
+
+from ...typing import Component
+from .helpers import extract_error, validate_formio_data
+
+
+class AddressNLValidationTests(SimpleTestCase):
+
+    def test_addressNL_field_required_validation(self):
+        component: Component = {
+            "key": "addressNl",
+            "type": "addressNL",
+            "label": "Required AddressNL",
+            "validate": {"required": True},
+        }
+
+        invalid_values = [
+            ({}, "required"),
+            ({"addressNl": ""}, "not_a_dict"),
+            ({"addressNl": None}, "null"),
+        ]
```

Comment:
I'm missing a test case like `{"addressNL": {"postcode": "1015CJ"}}` where some of the keys are missing and some present